protocols/overstimulateddao/overview.md

```diff
@@ -0,0 +1,42 @@
+## Overstimulate DAO by Daniel Allan
+
+
+Overstimulated comments on the overwhelming nature of my personal journey. When life gets hectic, my coping mechanism is music. Sometimes I don`t know how to communicate, so I suppose this is my way of sharing that experience with the world.
+
+## Motivation
+
+The decision to release Overstimulated as a community-owned DAO was simple.
+
+1. Retain creative control.
+1. Remove the creative bottleneck.
+1. Find a new avenue for building community.
+1. Experiment with a new way to fund a release.
+1. Raise working capital without selling ownership to a major label.
+1. Explore what a crypto-native EP looks like.
+
+This would be (to his knowledge) the first music crypto-funded crowdfund, which would offer great exposure to the space and encourage other artists to pursue independence.
+
+As he looks to make an impact in web3, He wants to explore a novel funding mechanism to truly engage fans and experiment with community governance.
+
+
+## Governance
+
+Overstimulated is community-owned, representing the master rights to the artist share of the project. The Mirror crowdfund raised 50 ETH, and backers received Overstimulated NFTs and $OVERSTIM tokens for their support.
+
+Overstimulated is a DAO governed by the $OVERSTIM token.
+
+The project (available on Discord) is a 6 track journey through my head.
+
+1. Stuck
+1. Feel Like That (with Idarose)
+1. Say What You Want (with Deegan)
+1. Really Cool (with Deegan)
+1. Poison (with DLG.)
+1. Too Close (with bloody white)
+
+
+## Additional Links
+
+[Website](https://danielallan.xyz/)
+[Twitter](https://twitter.com/imdanielallan)
+[Discord](https://discord.gg/mXQQfFT4Qt)
```

Comment:
Could you please add the snapshot URL?

Reply:
We usually don't add it as we are acting as UI for snapshot interface. And all the data pulled is from snapshot anyways.